.github/workflows/test-environment.yml

```diff
@@ -0,0 +1,102 @@
+name: sanity workflow
+
+on:
+  # Ability to execute on demand
+  workflow_dispatch:
+    inputs:
+      elk-stack-version:
+        required: true
+        description: "Elastic Cloud stack version"
+        default: "8.8.0-SNAPSHOT"
```

Comment:
I think you should find it by running `VERSION=$(grep defaultBeatVersion version/version.go | cut -f2 -d "\"")` to avoid maintaining this variable manually.

Reply:
@amirbenun in current context,  `elk-stack-version` is the input parameter for this workflow and the default version string will be displayed in the UI:
![Screenshot 2023-05-07 at 17 26 05](https://user-images.githubusercontent.com/99176494/236683980-03b88d1c-1bab-4443-8d72-d2e65875284c.png)

Setup job parses params and setup appropriate output vars. This parameter is defined by the user that triggers this workflow.
Your suggestion is relevant for scheduled workflow, when you need to get current latest version. But this option will be added later.